docs/sources/dashboards/build-dashboards/create-ng-dashboard/index.md

```diff
@@ -0,0 +1,373 @@
+1. (Optional) Enter a description of the changes you've made.
+1. Click **Save**.
+1. Toggle off the edit mode switch.
+
+## Edit dashboards
+
+When the dashboard is in edit mode, the edit pane that opens displays options associated with the part of the dashboard that it's in focus.
+For example, if you click in the area of a panel, row, or tab, that area comes into focus and the edit pane shows the options for that area:
+
+{{< figure src="/media/docs/grafana/dashboards/screenshot-edit-pane-focus-v12.png" max-width="750px" alt="Dashboard with a panel in focus" >}}
+
+- For rows and tabs, all of the available options are in the edit pane.
+- For panels, high-level options are in the edit pane and further configuration options are in the **Edit panel** view.
+- For dashboards, high-level options are in the edit pane and further configuration options are in the **Settings** page.
+
+To edit dashboards, follow these steps:
+
+1. Click **Dashboards** in the main menu.
+1. Navigate to the dashboard you want to update.
+1. Toggle on the edit mode switch.
+
+   The **Dashboard** edit pane opens on the right side of the dashboard.
+
+1. Click in the area you want to work with to bring it into focus and display the associated options in the edit pane.
+1. Do one of the following:
+
+   - For rows or tabs, make the required changes using the edit pane.
+   - For panels, update the panel title, description, repeat options or show/hide rules in the edit pane. For more changes, click **Configure** and continue in **Edit panel** view.
+   - For dashboards, update the dashboard title, description, grouping or panel layout. For more changes, click the settings (gear) icon in the top-right corner.
+
+1. When you've finished making changes, click **Save**.
+1. (Optional) Enter a description of the changes you've made.
+1. Click **Save**.
+1. Toggle off the edit mode switch.
+
+## Move or resize a panel
+
+<!-- previous headings Move a panel & Resize a panel -->
+
+When you're dashboard has a **Custom** layout, you can resize or move a panel to any location on the dashboard.
+
+To move or resize, follow these steps:
+
+1. Click **Dashboards** in the main menu.
+1. Navigate to the dashboard you want to update.
+1. Toggle on the edit mode switch.
+1. Do one of the following:
+
+   - Click the panel title and drag the panel to the new location.
+   - Click and drag the lower-right corner of the panel to change the size of the panel.
+
+1. Click **Save**.
+1. (Optional) Enter a description of the changes you've made.
+1. Click **Save**.
+1. Toggle off the edit mode switch.
+
+## Navigate using the dashboard outline
+
+The dashboard **Outline** provides a tree-like structure that shows you all of the parts of your dashboard and their relationships to each other including panels, rows, tabs, and variables.
+The outline also lets you quickly navigate the dashboard so that you don't have to spend time finding a particular element in order to work with it.
```

Comment:
```suggestion
The outline also lets you quickly navigate the dashboard so that you don't have to spend time finding a particular element to work with it.
```